### PR TITLE
[4.12] OCPBUGS-6789: enable pod security admission for techpreview

### PR DESCRIPTION
**Pull request by stlaz**
backport of https://github.com/openshift/cluster-kube-apiserver-operator/pull/1403

requires https://github.com/openshift/api/pull/1393

/cc @deads2k @ibihim

### CODE REVIEW

**Comment by stlaz** [comment]
/hold
contains fake bump

**Comment by deads2k** [comment]
/approve
/assign @ibihim 

for lgtm

File: bindata/assets/config/defaultconfig.yaml [hunk 1]
```diff
@@ -14,12 +14,12 @@ admission:
...
-          audit: "restricted"
-          audit-version: "latest"
-          warn: "restricted"
-          warn-version: "latest"
+          enforce: "invalid-to-force-substitution"
+          enforce-version: "invalid-to-force-substitution"
+          audit: "invalid-to-force-substitution"
+          audit-version: "invalid-to-force-substitution"
+          warn: "invalid-to-force-substitution"
+          warn-version: "invalid-to-force-substitution"
```
[the remark this answers is not in the document]
Comment (stlaz) [reply bindata/assets/config/defaultconfig.yaml:22]: It's to make sure that it always gets replaced

**Review by ibihim — APPROVED** [review]
/lgtm

**Pull request merged**